src/components/DraftbotBreakdown.js

```diff
@@ -210,45 +217,24 @@ export const Internal = ({ cardsInPack, draft, pack, picks, picked, seen }) => {
     return res;
   }, [cardsInPack]);
 
-  const { items, requestSort, sortConfig } = useSortableData(counts, { key: 'Total', direction: 'descending' });
-
   return (
     <>
       <Label check className="pl-2">
         <Input type="checkbox" onClick={toggleNormalized} /> Normalize the Columns
       </Label>
-      <Table bordered responsive className="small-table mt-lg-3">
-        <thead>
-          <tr>
-            <td />
-            {TRAITS.map((trait) => (
-              <HeaderCell
-                label={trait.name}
-                fieldName={trait.name}
-                sortConfig={sortConfig}
-                requestSort={requestSort}
-                tooltip={trait.description}
-              />
-            ))}
-          </tr>
-        </thead>
-        <tbody className="breakdown">
-          {items.map((item) => (
-            <tr key={item.card.details.cardID}>
-              <th scope="col">
-                <AutocardItem key={item.card.index} card={item.card} data-in-modal index={item.card.index}>
-                  <a href={`/tool/card/${encodeName(item.card.cardID)}`} target="_blank" rel="noopener noreferrer">
-                    {item.card.details.name}
-                  </a>
-                </AutocardItem>
-              </th>
-              {TRAITS.map((trait) => (
-                <td key={trait.name}>{item[trait.name]}</td>
-              ))}
-            </tr>
-          ))}
-        </tbody>
-      </Table>
+      <SortableTable
+        columnProps={TRAITS.map(({ name, description }) => ({
```

Comment:
The table displays all the data columns, but doesn't include names of the drafted cards in rows (see below).

![Screenshot from 2021-01-15 13-33-57](https://user-images.githubusercontent.com/38463785/104728066-4143b300-572e-11eb-9aac-7efb4c43ed09.png)